politeiawww/cmd/piwww/piwww.go

```diff
@@ -44,6 +44,12 @@ var (
 	errProposalMDNotFound = errors.New("proposal markdown file not " +
 		"found; you must either provide a markdown file or use the " +
 		"flag --random")
+	// errEditProposalRandomAndNameFound is emitted when both --name
+	// and --random flags found in editproposal command
+	errEditProposalRandomAndNameFound = errors.New("Please use either --random or --name")
```

Comment:
I typically will only initialize a error as a variable for one of two reasons:
1. You need to key off of it in some other part of the code.
2. The same error is being returned in multiple places. 

2 applies here, but if you're going to declare these you need to use them in the `newproposal` command as well.  I typically prefer nice short error messages like you have here, but CLI errors are a little different since there's a user on the other end of them.  Better to be verbose and tell the user exactly what they're doing wrong.  The error message you have here could cause some confusion.

Also, the golden rule of contributing to a codebase is to match the existing style.  The existing error messages don't capitalize the first letter so these shouldn't either.